packages/types/src/oauth/github.ts

```diff
@@ -38,4 +57,27 @@ export const GithubBearerTokenSchema = z.object({
   token_type: z.string(),
 });
 
-export type GithubBearerToken = z.infer<typeof GithubBearerTokenSchema>;
\ No newline at end of file
+export type GithubBearerToken = z.infer<typeof GithubBearerTokenSchema>;
+
+export const GithubUserResponseSchema = z.object({
+  login: z.string(),
+});
+
+export type GithubUserResponse = z.infer<typeof GithubUserResponseSchema>;
+
+export function MapGithubCommitContributionsToLeaderboardEntry(username: string, chipIssuer: ChipIssuer, resp: any): LeaderboardEntry {
+
+  let commits: number = 0;
+
+  if (resp && resp.data && resp.data.user && resp.data.user.contributionsCollection && resp.data.user.contributionsCollection.totalCommitContributions) {
+    commits = resp.data.user.contributionsCollection.totalCommitContributions;
+  }
```

Comment:
While this is currently only returning the commit count my query has other contribution info: 
```
{
    "data": {
        "user": {
            "contributionsCollection": {
                "totalCommitContributions": 2,
                "totalIssueContributions": 1,
                "totalPullRequestContributions": 8,
                "totalPullRequestReviewContributions": 7
            }
        }
    }
}
```

I had an oddly low commit count from the last week, but with all contributions it looks about right (2 commits vs ~20 contributions). I wonder if the low commit number has to do with squashing commits?